src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -4249,5 +4477,48 @@ private static void ValidateCosmosDbSetup(TestServer server)
 
             return false;
         }
+
+        private static string GetSimpleReadGraphQLQuery()
+        {
+            return @"{
+                    book_by_pk(id: 1) {
+                       id,
+                       title,
+                       publisher_id
+                    }
+                }";
+        }
+
+        private static string GetSimpleGraphQLCreateMutation()
+        {
+            return @"mutation createbook{
+                        createbook(item: { title: ""Book #1"", publisher_id: 1234 }) {
+                            title
+                            publisher_id
+                        }
+                    }";
+        }
+
+        private static string GetSimpleGraphQLIntrospectionQuery()
```

Comment:
does this really need to be a separate helper? it's only used once to demonstrate Query Depth for introspection and probably shouldn't be used in other tests because this is written to test specific settings you have for your introspection test.

Also makes reading the test harder because you have to skip around in this file to see the actual contents of the GraphQL request.